### PR TITLE
chore: Fix the release script

### LINKED ISSUE #2324: Fix the master build script
**Describe the chore**
The script in the master build is failing. It needs to be fixed

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by atm1504**
Fixes #2324 

Changes: 
Updated the script according to the one present in : https://github.com/fossasia/open-event-organizer-android/pull/1770

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -21,5 +21,10 @@ allprojects {
     repositories {
         google()
     }
+
+    ext {
+        KEYSTORE_FILE = rootProject.file('scripts/key.jks')
+        CIRCLE_BUILD = System.getenv("CIRCLE") == "true" && KEYSTORE_FILE.exists()
```
Comment (iamareebjamal) [inline build.gradle]: Show me where does it say that CircleCI Builds have `CIRCLE` as an environment variable

**Comment by iamareebjamal** [comment]
You also have to run prep-key before the build. Please see the reference PR carefully

File: build.gradle (hunk 1 above)
Comment (atm1504) [reply build.gradle]: Changed it to ```CIRCLECI```. This is present in the documentation.

**Comment by atm1504** [comment]
> You also have to run prep-key before the build. Please see the reference PR carefully

Updated the circleci script. ```prep-key``` is now executed before ```./gradlew build```

**Review by iamareebjamal — APPROVED** [review]

File: build.gradle [hunk 2]
```diff
@@ -21,5 +21,9 @@ allprojects {
     repositories {
         google()
     }
-}
 
+    ext {
+        KEYSTORE_FILE = rootProject.file('scripts/key.jks')
```
Comment (iamareebjamal) [inline build.gradle:26]: This should have been exec/key.jks

Comment (atm1504) [reply build.gradle:26]: In the open event repo, there is a folder called exec, inside which all the files were kept. But here we have a folder named scripts, where such files are kept. So I used scripts there.

 Can you please tell what's the use of the above statement.

Comment (iamareebjamal) [reply build.gradle:26]: You are saying completely opposite things. open event repo has scripts folder and that's why scripts was used and susi has exec folder. You didn't change anything. You just copied. You should have changed this to exec

**Pull request merged**